src/es_client.ts

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable require-jsdoc */
+import {Client} from '@elastic/elasticsearch';
+
+interface clientOptions {
+    node: string,
+    username: string,
+    password: string,
+}
+
+
+export function initClient(options: clientOptions) {
+    const client = new Client({
+        node: options.node,
+        auth: {
+            username: options.username,
+            password: options.password
+          }
+    });
+
+    return {
+        getTransactions: async function(jobId: string, startTime: string){
```

Comment:
For reference if you wanna check about previous comment https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-simple-query-string-query.html

Reply:
As discussed, let's leave it for later. We don't need it in POC